swingset/src/main/java/com/nqadmin/swingset/SSTextField.java

```diff
@@ -478,10 +494,13 @@ public void setNumberOfDecimalPlaces(final int _numberOfDecimalPlaces) {
 	 * Sets the SSCommon data member for the current Swingset Component.
 	 *
 	 * @param _ssCommon shared/common SwingSet component data and methods
+	 * @deprecated Creates inconsistent state, no replacement.
 	 */
+	// TODO: is this used? What about all the listeners and such?
```

Comment:
I think SSComponentInterface() could be modified to remove setSSCommon(). I did a reference search in Eclipse and its not used from within SwingSet. I also don't see references in our software using SwingSet.

I don't see any reason ssCommon would need to be replaced once a SwingSet component is instantiated.

Best guess here is that when I created SSCommon and SSComponentInterface, I was creating setter/getter pairs to be thorough, but I agree that it's dangerous to call setSSCommon().